AirCasting/CreateSessionViews/TurnOnBluetoothView.swift

```diff
@@ -11,10 +11,12 @@ import SwiftUI
 
 struct TurnOnBluetoothView: View {
     @State private var isPowerABLinkActive = false
+    @State private var SDRestartABLink = false
     @EnvironmentObject var settingsRedirection: DefaultSettingsRedirection
     @EnvironmentObject var bluetoothManager: BluetoothManager
     @Binding var creatingSessionFlowContinues: Bool
     @Binding var sdSyncContinues: Bool
+    var isSDClearProcess: Bool
```

Comment:
Maybe this could also default to "false"? This way we wouldn't have to set it to false in all of those views above.